deployment/ansible/playbook.yml

```diff
@@ -8,7 +8,7 @@
 
   roles:
     - { role: "azavea.terraform" }
-    - { role: "azavea.java" } 
+    # - { role: "azavea.java" } 
```

Comment:
How come this is commented out instead of being removed?

Reply:
Not sure - @tnation14 is there any reason to leave this in? We were just doing that to test it out, but perhaps it should be removed.

Reply:
No, this is safe to remove.